library/include/borealis/i18n.hpp

```diff
@@ -64,6 +64,14 @@ void loadTranslations();
  */
 std::string getCurrentLocale();
 
+#ifdef __SWITCH__
```

Comment:
Why put it here? Technically it's i18n related but it doesn't belong to the i18n system

Reply:
I mean it depends on the Switch implementation, and i18n is supposed to be generic (and eventually will)

Reply:
this id only make sense for libnx i think
removed `#ifdef`